modules/module-list.nix

```diff
@@ -10,10 +12,9 @@
   ./environment/path.nix
   ./environment/session-init.nix
   ./home-manager.nix
-  ./nixpkgs.nix
   ./time.nix
   ./user.nix
   ./version.nix
   ./workaround-make.nix
-  <nixpkgs/nixos/modules/misc/assertions.nix>
-]
+  (pkgs.path + "/nixos/modules/misc/assertions.nix")
+] ++ pkgs.lib.optionals (!isFlake) [ ./nixpkgs.nix ]
```

Comment:
```suggestion
] ++ pkgs.lib.optional (!isFlake) ./nixpkgs.nix
```

Reply:
Yeah I used `optionals` because of https://discourse.nixos.org/t/lib-optional-considered-harmful/7081 but I'll make the change.

Reply:
Oh good point, you can leave it like that :)